webserver/server/app/services/utils.ts

```diff
@@ -0,0 +1,40 @@
+import type { Asset } from '../../../shared/models/common';
+import { Dex } from '../../../shared/models/common';
+
+export function parseAssetItem(s: string | undefined | null): Buffer {
+    // For the sake of the query, we represent ADA as ('', '') instead of (NULL, NULL).
+    // (see sqlDexMeanPrice.queries.sql for details)
+    return Buffer.from(s ?? "", 'hex');
+  }
+  
+  export function serializeAsset(policyId: Buffer | null, assetName: Buffer | null): Asset {
+    if (policyId === null && assetName === null) {
+      return null;
+    }
+    if (policyId !== null && assetName !== null) {
+      return {
+        policyId: policyId.toString('hex'),
+        assetName: assetName.toString('hex'),
+      };
+    }
+    throw new Error('Invalid asset query response'); // should be unreachable
+  }
+  
+
+export function valueToDex(dex: string) {
+    switch(dex) {
+        case '0': return Dex.WingRiders;
+        case '1': return Dex.SundaeSwap;
+        case '2': return Dex.MinSwap;
+    }
+    return Dex.Unknown;
+}
+
+export function dexToValue(dex: Dex) {
+    switch(dex) {
+        case Dex.WingRiders: return '0';
+        case Dex.SundaeSwap: return '1';
+        case Dex.MinSwap: return '2';
+    }
+    return '-1';
```

Comment:
Maybe `Dex` should not be a string-based enum if we actually care about converting to/from numeric values this much

Reply:
To deal only with numbers is totally better, but the enum is more readable without special documentation/mapping